src/g_bang.c

```diff
@@ -75,6 +77,9 @@ void bng_draw_new(t_bng *x, t_glist *glist)
              (strcmp(x->x_gui.x_lab->s_name, "empty") ? x->x_gui.x_lab->s_name : ""),
              x->x_gui.x_font, x->x_gui.x_fontsize * IEMGUI_ZOOM(x), sys_fontweight,
              x->x_gui.x_lcol, x);
+#ifdef HAVE_KEYBOARDNAV
```

Comment:
the `HAVE_KEYBOARDNAV` is totally pointless, as it only clutters the code and adds more compile-paths, making it much more difficult to find errors.

if you this is an attempt to keep binary-compatibility, then it didn't work
- you've already broken the (private) ABI by adding a new member to the `t_editor` struct. But now that this is done, you can add whatever you want to `t_editor_private` **without** having to worry about ABI.
- even if `HAVE_KEYBOARDNAV` had an impact on the ABI, this would be bad design. we don't want to ABI to change depending on the compile-time options one choses (but luckily the define doesn't change the ABI anyhow)
- what's the point of a compile-time option that cannot be changed? if you want it settable, you should also add an option to the autotools system (but please don't)

so: please get rid of all the `#ifdef HAVE_KEYBOARDNAV` protectors.

Reply:
I'm gonna resolve this as i've put this in the TODO in the first post.